extensions/arc/src/common/kubeUtils.ts

```diff
@@ -0,0 +1,48 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as path from 'path';
+import * as os from 'os';
+import * as yamljs from 'yamljs';
+import * as fs from 'fs';
+
+export interface KubeClusterContext {
+	name: string;
+	isCurrentContext: boolean;
+}
+
+export async function getKubeConfigClusterContexts(configFile: string): Promise<KubeClusterContext[]> {
+	try {
+		await fs.promises.access(configFile);
+	} catch (error) {
+		if (error && error.code === 'ENOENT') {
+			return [];
```

Comment:
I agree, since we are already throwing when it is not access denied, I do not see the need to special case. No sure why @alanrenmsft, special cased it originally in rd.

Reply:
remove special checking for ENOENT, we will just bubble up any error while accessing the config file.